packages/web/src/scss/components/Breadcrumbs/_Breadcrumbs.scss

```diff
@@ -0,0 +1,80 @@
+@use 'sass:map';
+@use 'theme';
+@use '../../tools/breakpoint';
+@use '../../tools/svg';
+@use '../../tools/typography';
+
+.Breadcrumbs {
+    @include typography.generate(theme.$typography);
+
+    width: 100%;
+}
+
+.Breadcrumbs > ol {
+    display: flex;
+    flex-wrap: wrap;
+    gap: theme.$gap-mobile-up;
+    padding: 0;
+    margin: 0;
+    list-style: none;
+
+    @include breakpoint.up(map.get(theme.$breakpoints, tablet)) {
+        gap: theme.$gap-tablet-up;
+    }
+}
+
+.Breadcrumbs > ol > li {
+    display: flex;
+    align-items: center;
+    gap: theme.$gap-mobile-up;
+
+    @include breakpoint.up(map.get(theme.$breakpoints, tablet)) {
+        gap: theme.$gap-tablet-up;
+    }
+
+    &::before {
+        content: '';
+        display: inline-block;
+        flex-shrink: 0;
+        width: theme.$icon-width;
+        height: theme.$icon-height;
+        background: svg.escape(theme.$icon-chevron-left) no-repeat;
+
+        @include breakpoint.up(map.get(theme.$breakpoints, tablet)) {
+            background: svg.escape(theme.$icon-chevron-right) no-repeat;
+        }
+    }
+
+    &:nth-of-type(2)::before {
+        @include breakpoint.up(map.get(theme.$breakpoints, tablet)) {
+            content: none;
+        }
+    }
+}
+
+.Breadcrumbs > ol > li > a {
+    text-decoration: underline;
+    color: theme.$link-color-default;
+
+    &:active {
+        text-decoration: none;
```

Comment:
Are you sure? 🤔 I think the decoration should be always there because it applies the `Body/Small/Link-Underline` style.

In helpers the style looks [like this](https://spirit.supernova-docs.io/spirit/latest/development/helpers.html#section-examples-b7).

<img width="739" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/185185871-54727adb-47b3-42dd-8c31-b1f97d0e523e.png">